test/CMakeLists.txt

```diff
@@ -54,6 +54,9 @@ addtest(readonly_direct)
 addtest(sanitize_blacklist)
 addtest(secondary_file)
 addtest(secondary_http)
+if(HTTPS_STORAGE_BACKEND)
```

Comment:
This condition is ignored by direct invocation of `run` as called by `build-and-verify-package`. One way to fix it would be to teach ccache about all its optional features (may reported by `--version`) and then have `_PROBE` skipping the test suite if a required feature is not present.

Reply:
Yes, adding a list of optional features to `--version` sounds good.

Reply:
Merged #897